front/app/modules/commercial/content_builder/services/ContentBuilder.ts

```diff
@@ -0,0 +1,48 @@
+import { API_PATH } from 'containers/App/constants';
+import streams from 'utils/streams';
+
+export interface IBuilderLayoutData {
+  type: 'project';
+  id: string;
+  attributes: {
+    craftjs_jsonmultiloc: JSON;
+    code: string;
+    enabled: boolean;
+    created_at: string;
+    updated_at: string;
+  };
+}
+
+export interface IBuilderLayout {
+  data: IBuilderLayoutData;
+}
+
+export function builderLayoutStream(projectId: string, code: string) {
+  return streams.get<IBuilderLayout>({
+    apiEndpoint: `${API_PATH}/projects/${projectId}/content_builder_layouts/${code}`,
+  });
+}
+
+export function updateBuilderLayout(
+  projectId: string,
+  code: string,
+  dataId: string,
+  object: IBuilderLayout
```

Comment:
I think we need a different type for the request object - it won't be exactly the same as the response - won't have `id` for example. Maybe check with Koen what is expected that we send in the request body.

Reply:
@IvaKop Would the id be the project ID maybe? Or should I remove that ID altogether as it's included in the URL?

Reply:
You can remove it as it's not needed.